manager/manager.go

```diff
@@ -183,7 +190,7 @@ func New(memoryCache *memory.InMemoryCache, sysfs sysfs.SysFs, maxHousekeepingIn
 	// Register for new subcontainers.
 	eventsChannel := make(chan watcher.ContainerEvent, 16)
 
-	newManager := &manager{
+	newManager = &manager{
```

Comment:
It's better move those definitions to L155 and only assign `fsInfo` and `eventsChannel` here.